File: main/src/main/java/net/fwitz/math/main/binary/split/functions/hyperbolic/SplitTanhZ.java
package net.fwitz.math.main.binary.split.functions.hyperbolic;

import net.fwitz.math.binary.split.SplitComplex;
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot;

public class SplitTanhZ {
    public static void main(String[] args) {
        new SplitComplexFunctionPlot("tanh(z)")
                .computeFn(SplitComplex::tanh)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/split/functions/hyperbolic/SplitTanhZ.kt
package net.fwitz.math.main.binary.split.functions.hyperbolic

import net.fwitz.math.binary.split.SplitComplex
import net.fwitz.math.plot.binary.split.SplitComplexFunctionPlot

object SplitTanhZ {
    @JvmStatic
    fun main(args: Array<String>) = SplitComplexFunctionPlot("tanh(z)")
        .computeFn(SplitComplex::tanh)
        .render()
}